app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileResetPinActivityTest.kt

```diff
@@ -395,7 +395,7 @@ class ProfileResetPinActivityTest {
   }
 
   @Test
-  fun testProfileResetPinActivity_startActivityWithAdmin_inputWrongConfirmPin_clickSave_checkConfirmWrongError() { // ktlint-disable max-line-length
+  fun testProfileResetPin_withAdmin_inputWrongConfirmPin_save_checkConfirmWrongErrorIsShown() {
```

Comment:
testProfileResetPin_withAdmin_inputWrongConfirmPin_save_confirmWrongErrorIsShown

Reply:
Fixed, thank you.